### PR TITLE
Break up the examples page into 4 pages

### LINKED ISSUE #904: Break up the examples page into 4 pages
We should break up the examples page into 4 separate pages (one for each section on the page).
https://www.kubeflow.org/docs/examples/resources/

Also add an _index page, following the pattern of the other doc directories.

This will solve 2 problems:

* The examples page is getting a little long.
* The link to the examples page in the left-hand nav looks odd, because all other links are to sections and are therefore bold.

### PR DESCRIPTION
**Pull request by mameshini**
fixes #904

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/922)

### CODE REVIEW

File: content/docs/examples/kubeflow-samples.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
++++
+title = "Kubeflow Samples"
+description = "Examples that demonstrate machine learning with Kubeflow"
+weight = 10
++++
+
+{{% blocks/content-item %}}
```
Comment (joeliedtke) [inline content/docs/examples/kubeflow-samples.md]: Please remove the {{% blocks/content-item %}} and {{% /blocks/content-item %}} shortcodes for this sentence.

Comment (mameshini) [reply content/docs/examples/kubeflow-samples.md]: Good catch, I removed it

**Pull request merged**